src/enable_variable_ops/ngraph_rewrite_for_tracking.cc

```diff
@@ -67,29 +69,36 @@ Status RewriteForTracking(Graph* graph, int graph_id) {
       for (auto edge : node->out_edges()) {
         if (edge->dst()->IsOp() && !edge->IsControlEdge() &&
             IsRefType(edge->dst()->input_type(edge->dst_input()))) {
+          just_looking = false;
           // if the output reference is read by NGraph supported ops, do not
-          // turn off just_looking
+          // turn off is_tf_modifying
           if (!IsNGVariableType(edge->dst()->type_string())) {
-            NGRAPH_VLOG(1) << DebugNode(edge->dst())
-                           << "needs reference, setting just_looking to false";
-            just_looking = false;
+            NGRAPH_VLOG(1)
+                << DebugNode(edge->dst())
+                << "needs reference, setting is_tf_modifying to false";
+            is_tf_modifying = false;
```

Comment:
Either change the name 'is_tf_not_modifying' or  set  'is_tf_modifying = true'
In the 2nd case you will need to change the checks for (!is_tf_modifying) to (is_tf_modifying)

Reply:
Changed it to is_tf_just_looking